### DISCUSSION TITLE
Run trino-openapi with Petstore v2

### DISCUSSION THREAD

**Discussion by shoaibahmed9922** [nineinchnick/trino-openapi#117] [Q&A]
I'm starting a docker container using below

```
docker run \
  -d \
  --name trino-openapi \
  -e OPENAPI_SPEC_LOCATION=https://petstore.swagger.io/v2/swagger.json \
  -e OPENAPI_BASE_URI=petstore.swagger.io/v2 \
  -p 8080:8080 \
  nineinchnick/trino-openapi:1.39
```

Container exits with the following error

```
Unable to create injector, see the following errors:
2024-08-29 16:57:02 
2024-08-29 16:57:02 1) [Guice/ErrorInjectingConstructor]: IllegalArgumentException: Failed to parse the OpenAPI spec: attribute openapi is missing
```

Am I missing something?
If I am not doing it right then please share `docker run` command to start `trino-api` container with OPENAPI_PETSTORE enabled.
Thanks

**Comment by nineinchnick** [comment]
OpenAPI v2 is not supported, only OpenAPI v3. Try using https://petstore3.swagger.io/api/v3/openapi.json

**Reply by shoaibahmed9922** [reply]
It worked like a charm but still I'm not able to run below query.

`SELECT * FROM  openapi.default.pet_find_by_status where status='available'`

It is raising below error

`Query failed: Expected response code to be [200, 201, 202, 203, 204, 205, 206], but was 401`

**Reply by nineinchnick** [reply]
401 means you're not authenticated. Check your catalog configuration.
